src/app/components/Tokens/TokenTransfers.tsx

```diff
@@ -82,6 +129,7 @@ type TokenTransfersProps = {
   isLoading: boolean
   limit: number
   pagination: false | TablePaginationProps
+  tickersAsLink?: boolean | undefined
```

Comment:
`tickersAsLink:`

Reply:
I would prefer not doing that, because I want to able to use this component without specifying this optional prop. Removing the `?` make that impossible.